src/picongpu/include/fields/laserProfiles/laserPolynom.hpp

```diff
@@ -59,7 +59,7 @@ HINLINE float3_X laserLongitudinal(uint32_t currentStep, float_X& phase)
     elong.x() = float_X(
                        float_64(AMPLITUDE) *
```

Comment:
I just added it because the same assignment uses some mixing of doubles and floats below.